cli/cmd/plugin/unmanaged-cluster/cluster/minikube.go

```diff
@@ -0,0 +1,310 @@
+// Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cluster
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/vmware-tanzu/community-edition/cli/cmd/plugin/unmanaged-cluster/config"
+
+	"k8s.io/klog/v2"
+	minikubeConfig "k8s.io/minikube/pkg/minikube/config"
```

Comment:
@jpmcb I noticed this brings in **a lot** of transitive (indirect) dependencies. And throughout we're using `os.Exec` to call minikube with this exception for `ListProfiles`.

For the sake of consistency and **not** importing `minikube`, I think we'd be better off using and serializing against:

```
$ minikube profile list -o json
{"invalid":[],"valid":[]}
```